convert2rhel/unit_tests/pkgmanager/handlers/yum_test.py

```diff
@@ -180,6 +180,9 @@ def test_process_transaction_with_exceptions(self, pretend_os, _mock_yum_api_cal
             pkgmanager.Errors.YumRPMCheckError,
             pkgmanager.Errors.YumTestTransactionError,
             pkgmanager.Errors.YumRPMTransError,
+            pkgmanager.Errors.YumDownloadError,
+            pkgmanager.Errors.YumBaseError,
+            pkgmanager.Errors.YumGPGCheckError,
         )
```

Comment:
Do the side_effects need to be given to this unit test via parametrize?  At first glance, it looks like this unit test will only tun once, raising `YumRPMCheckError` and checking that will lead to us calling `loginst.critical()`.

Reply:
We can simplify that to raising only the YumBaseError